### DISCUSSION TITLE
Pages 404 Error

### DISCUSSION THREAD

**Discussion by lukeiscoolio** [community/community#23510] [Other Feature Feedback, Questions, & Ideas]
<p>Hello! I am trying to make a github pages, but I keep getting this error<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/8/f8b394e2cce8ceb3a520fc0bdab447f57ba1eeed.png" data-download-href="/uploads/short-url/zu7ac7FNFv3fQkZBVigDrx8YpEp.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/98642495/181077470-4af18835-05b0-4c35-ab2a-ee6712b97d80.png" alt="image" data-base62-sha1="zu7ac7FNFv3fQkZBVigDrx8YpEp" width="690" height="403" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/8/f8b394e2cce8ceb3a520fc0bdab447f57ba1eeed_2_690x403.png, https://aws1.discourse-cdn.com/github/original/3X/f/8/f8b394e2cce8ceb3a520fc0bdab447f57ba1eeed.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/8/f8b394e2cce8ceb3a520fc0bdab447f57ba1eeed.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/8/f8b394e2cce8ceb3a520fc0bdab447f57ba1eeed_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">936×547 20 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
I have put in index.html with this code
<pre><code class="lang-auto">&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;body&gt;
&lt;h1&gt;Hello World&lt;/h1&gt;
&lt;p&gt;I'm hosted with GitHub Pages.&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;
</code></pre>
<p>Here is the repo <a href="https://github.com/SunsetLodge/support-website" class="inline-onebox" rel="noopener nofollow ugc">GitHub - SunsetLodge/support-website</a> and the pages <a href="https://sunsetlodge.github.io/support-website/" rel="noopener nofollow ugc">https://sunsetlodge.github.io/support-website/</a></p>
<p>I thought it had to do something with this.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/6/36026803ca73ff37af5554b29e95094c619a952e.png" data-download-href="/uploads/short-url/7HMWd6YnLWe2IXdO4ZV31pXOixE.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/98642495/181077471-c130ad43-d1f7-4053-aa2e-0ad3193783bd.png" alt="image" data-base62-sha1="7HMWd6YnLWe2IXdO4ZV31pXOixE" width="690" height="138" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/6/36026803ca73ff37af5554b29e95094c619a952e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">698×140 6.85 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s now working</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077472-5dd500de-8bd8-4fdb-ab9f-dd5fcfc81040.png" alt="image" data-base62-sha1="3Erz8JzWQFVb7xTZy1dZNDtDa2m" width="449" height="228"></p>

**Comment by lukeiscoolio** [comment]
<p>Hm, I wonder why it wasn’t working before.</p>

**Comment by jfullstackdev** [comment]
<p>usually, it will take some time,</p>
